docs/file.md

```diff
@@ -0,0 +1,88 @@
+Chunked Upload
+--------------
+
+For large files or in cases where the network connection is less reliable,
+you may want to upload the file in parts.  This allows a single part to fail
+without aborting the entire upload, and failed parts can then be retried.
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+
+- [Manual Process](#manual-process)
+  - [Create Upload Session for File Version](#create-upload-session-for-file-version)
+  - [Create Upload Session for File](#create-upload-session-for-file)
+  - [Upload Part](#upload-part)
+  - [Commit Upload Session](#commit-upload-session)
+  - [Abort Upload Session](#abort-upload-session)
+  - [List Upload Parts](#list-upload-parts)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+### Manual Process
+
+For more complicated upload scenarios, such as those being coordinated across
```

Comment:
This paragraph doesn't make sense. The manual process is the only process implemented.